modules/core/src/test/java/org/apache/ignite/internal/processors/cache/persistence/db/filename/IgniteUidAsConsistentIdMigrationTest.java

```diff
@@ -587,6 +589,47 @@ public void testStartOldStyleNoPortsNodesByCompatibleProperty() throws Exception
         System.clearProperty(IGNITE_CONSISTENT_ID_BY_HOST_WITHOUT_PORT);
     }
 
+    /**
+     * Test start node with defined consistent id through system property.
+     * Expected to be 1 folder with specified name.
+     *
+     * @throws Exception if failed.
+     */
+    @Test
+    public void testStartNodeWithDefinedConsistentIdBySystemProperty() throws Exception {
```

Comment:
To be honest, I don't like having the test here.
It is a migration test comparing the old and new consistent ID generation when it is not set which is different from testing how it can be set explicitly.

Can you add a separate test with the following cases
- consistentID is not set in config, not set as a property
- consistentID is set in config, not set as a property
- consistentID is not set in config, set as a property
- consistentID is set in config, set as a property

In each case check that the resulting consistentID is as expected. In the first case it will be just a random UUID (you can check it via regexp, for example). In the second case it will be the one from the config. In the third and fourth it will be the one from the property.